app/src/view/screens/PostLetter/PostLetterInfo.js

```
@@ -800,126 +799,135 @@ export default function PostLetterInfo({ navigation }) {
-              justifyContent: 'space-evenly',
+              height: hp(0.1),
+              marginHorizontal: wp(8),
               marginTop: hp(3),
-            }}>
-            <TouchableOpacity
-              onPress={() => setLetterType('Public Letter')}
-              style={{ flexDirection: 'row', marginHorizontal: wp(7) }}>
-              <PublicLetter height={23} width={23} />
+              backgroundColor: "#00000012",
+            }}
+          ></View>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)